### ISSUE TITLE
Suddenly, all GoVee devices do no longer show up

### ISSUE THREAD

**Issue by 3Fish** [wez/govee2mqtt#189]
### Govee Device SKU

H605C, H6046, H6072, H61E1

### Govee2MQTT Version

2024.04.29-30cf7732

### Describe the issue

I discovered that my previously working setup did not receive any new updates from my GoVee hardware. I therefore updated everything to the newest version (GoVee devices and HA software) and tried again. No changes, I then thought maybe I needed to reinstall everything, so I removed the integration for mQTT and its devices and reinstalled both Mosquito and Govee2MQTT. MQTT works fine (tested with other devices) but I don't receive any updates from my Govee devices.
What I tried:
* ensure that the LAN API is enabled on all of them
* give each device a static IP and add it to the "Additional IPs" list
* tried "Send discovery to each network interface"
* tried "Send discovery to global broadcast address"
* setting a MQTT user & password

My network consists of two AVM Fritzboxes where one is configured as a mesh slave to the other one. All Goovee devices connect via Wi-Fi to the slave device. I checked the Fritzbox manual, but there is nothing regarding Multicasting or Broadcasting.

### Startup Diagnostics

s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
GOVEE_API_KEY=REDACTED
GOVEE_LAN_BROADCAST_GLOBAL=false
GOVEE_MQTT_HOST=core-mosquitto
GOVEE_TEMPERATURE_SCALE=C
GOVEE_LAN_SCAN=192.168.178.106,192.168.178.105,192.168.178.104,192.168.178.98,192.168.178.108,192.168.178.87
GOVEE_EMAIL=REDACTED
GOVEE_PASSWORD=REDACTED
GOVEE_MQTT_PASSWORD=REDACTED
GOVEE_LAN_BROADCAST_ALL=false
GOVEE_MQTT_USER=addons
GOVEE_LAN_NO_MULTICAST=false
GOVEE_MQTT_PORT=1883
++ cd /app
++ exec /app/govee serve
[2024-05-09T21:14:50 INFO  govee::commands::serve] Starting service. version 2024.04.29-30cf7732
[2024-05-09T21:14:50 INFO  govee::commands::serve] Querying platform API for device list
[2024-05-09T21:14:52 INFO  govee::commands::serve] Querying undocumented API for device + room list

**Comment by muzammilk** [comment]
I had the same but it started working on restarting govee2mqtt addon. Have you tried restarting?

**Comment by 3Fish** [comment]
@muzammilk , yes I restarted the add-on and HA multiple times. No improvement.

**Comment by 3Fish** [comment]
Seams to be the same issue as this
https://github.com/wez/govee2mqtt/issues/419
